packages/components/src/LightBox/LightBox.css

```diff
@@ -18,3 +18,61 @@
 .wrapper:global(.ril__outer .ril__toolbarItem) {
   color: var(--color-greyBlue--lighter);
 }
+
+.next,
+.prev {
+  display: flex;
+  position: absolute;
+  top: calc(50% - 20px);
+  z-index: 2;
+  width: 20px;
```

Comment:
These proportions are very random-seeming. Can we make them square? Would also like to round the corners (will add those comments on the next and prev classes below)

Reply:
![image](https://user-images.githubusercontent.com/39704901/222491959-314fcdf7-a657-40fa-8fe8-096b4319a52d.png)